patches/tModLoader/Terraria/Chest.cs.patch

```diff
@@ -139,28 +140,24 @@
  			}
  
  			item[num2] = newItem.Clone();
-@@ -529,6 +_,7 @@
+@@ -626,6 +_,7 @@
  			item[num2].stack -= num;
  			_ = item[num2].value;
  			_ = 0;
 +			return num2;
  		}
  
- 		public static void SetupTravelShop() {
-@@ -902,9 +_,12 @@
- 							Main.travelShop[num2++] = 3636;
- 							Main.travelShop[num2++] = 3641;
- 							break;
+ 		public static void SetupTravelShop_AddToShop(int it, ref int added, ref int count) {
+@@ -702,6 +_,8 @@
```

Comment:
I tried to match the code lines as close as possible to how it was in 1.3.6, but this specific code block was refactored quite a bit between the versions. It now rests about 200 files before where it previously rested.

I am not super well versed on the modloader so `NPCLoader.SetupTravelShop()` may need to be moved to another location, but for now it resides in `SetupTravelShop_AddToShop()` method.